src/hdnode.js

```diff
@@ -291,6 +291,30 @@ HDNode.prototype.isNeutered = function () {
   return !(this.keyPair.d)
 }
 
+HDNode.prototype.derivePath = function (path) {
+  typeforce(types.String, path)
+
+  var splitPath = path.split('/')
+  if (splitPath[0] === 'm') {
+    if (!this.parentFingerprint) {
+      splitPath = splitPath.slice(1)
+    } else {
+      throw new Error('Not a master node')
+    }
```

Comment:
I am not sure standard allow if block without curly braces :) but yeah I got you. Ok, will do tomorrow